quiche/src/recovery/congestion/bbr/per_ack.rs

```diff
@@ -258,13 +258,28 @@ fn bbr_check_drain(r: &mut Congestion, bytes_in_flight: usize, now: Instant) {
     }
 
     if r.bbr_state.state == BBRStateMachine::Drain &&
-        bytes_in_flight <= bbr_inflight(r, 1.0)
+        bbr_bytes_in_net(r, bytes_in_flight, now) <= bbr_inflight(r, 1.0)
     {
         // we estimate queue is drained
         bbr_enter_probe_bw(r, now);
     }
 }
 
+fn bbr_bytes_in_net(r: &Congestion, in_flight: usize, now: Instant) -> usize {
+    let edt = r.pacer.next_time().max(now);
+    let interval = edt.saturating_duration_since(now);
+    let interval_delivered = interval.as_secs_f64() * r.bbr_state.btlbw as f64;
```

Comment:
I cant tell but is `r.bbr_state.btlbw ` equivalent to `bbr_bw`. If its different should we call out the difference in a comment?

Reply:
I think that r.bbr_state.btlbw is equivalent to [lt_bw](https://elixir.bootlin.com/linux/v6.13.7/C/ident/lt_bw) in the kernel code, so bbr_bw would match btlbw when [lt_use_bw](https://elixir.bootlin.com/linux/v6.13.7/C/ident/lt_use_bw) is true.

I assume that we have a valid btlbw at this point, but it wouldn't hurt to double check (btlbw starts as 0)

Reply:
From what I can tell `btlbw` is the closest thing to `bw` in [BBR1 state](https://github.com/cloudflare/quiche/blob/951e1fce3267c62177a5f308f0cc1f9b286d6600/quiche/src/recovery/congestion/bbr/mod.rs#L103).

The RFC only seems to mention [BBR.BtlBw](https://datatracker.ietf.org/doc/html/draft-cardwell-iccrg-bbr-congestion-control-00#section-4.1.1).